source/extensions/listener_managers/listener_manager/listener_manager_impl.cc

```diff
@@ -211,6 +212,64 @@ ProdListenerComponentFactory::createUdpListenerFilterFactoryListImpl(
   return ret;
 }
 
+Filter::QuicListenerFilterFactoriesList
+ProdListenerComponentFactory::createQuicListenerFilterFactoryListImpl(
+    const Protobuf::RepeatedPtrField<envoy::config::listener::v3::ListenerFilter>& filters,
+    Configuration::ListenerFactoryContext& context,
+    Filter::QuicListenerFilterConfigProviderManagerImpl& config_provider_manager) {
+  Filter::QuicListenerFilterFactoriesList ret;
+
+  ret.reserve(filters.size());
+  for (ssize_t i = 0; i < filters.size(); i++) {
+    const auto& proto_config = filters[i];
```

Comment:
nit: there's an awful lot of auto here. The C++ style guide suggests only using auto when the type is obvious or for things like iterators where the type is horrific :) Some of these autos are protos so maybe it's ok, but consider using explicit types.